resources/assets/components/PagingButtons/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class PagingButtons extends React.Component {
+
+  render() {
+    const prev = this.props.prev;
+    const next = this.props.next;
+    const prevCopy = this.props.prevCopy;
+    const nextCopy = this.props.nextCopy;
+
+    return (
+        <div className="container__block">
+			<a href={prev}>{prevCopy}</a>
+			<div className="next-page">
```

Comment:
Should we check if `next` or `prev` are set here? We probably want to disable or hide the link if the user is on the first or last page.